app/views/players/edit.html.slim

```diff
@@ -2,16 +2,7 @@ h1.title
   | EDIT PLAYER
 .form
   = form_for :player, url: player_path(@player), method: :patch do |f|
```

Comment:
You can pass variables into the partial: `= render 'form', locals: { player: @player }`

Then in your `_form.html.slim`:
```slim
= form_for player do |f|
  .row
    span.form-label
      ...
```

Actually if you pass in the correct player, rails will try to deduce the url and HTTP method for you. You can read up at [this link](http://apidock.com/rails/ActionView/Helpers/FormHelper/form_for), search for 'Resource-oriented style'. 

cc. @moexmen/interns - For learning as well :+1: